src/test/java/Homework22.java

```diff
@@ -0,0 +1,28 @@
+import org.testng.Assert;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.Test;
+import pages.HomePage;
+import pages.LoginPage;
+
+public class Homework22 extends BaseTest {
+        String newPlayListName = "New playlist";
+        String currentPlayListName = "TestPro Playlist";
+        HomePage homePage = null;
+
+        @Test
+        public void renamePlayListTest() throws InterruptedException {
```

Comment:
Please, delete `throws InterruptedException`

Thread. sleep is bad! It blocks the current thread and renders it unusable for further work.
Using Thread. sleep() frequently in an automation framework is not a good practice.

Please use the Explicit Wait command, the WebDriver is directed to wait until a certain condition occurs before proceeding with executing the code.